include/SFML/Window/Mouse.hpp

```diff
@@ -46,32 +46,33 @@ namespace Mouse
 /// \brief Mouse buttons
 ///
 ////////////////////////////////////////////////////////////
-enum Button
+enum class Button
 {
     Left,     //!< The left mouse button
     Right,    //!< The right mouse button
     Middle,   //!< The middle (wheel) mouse button
     XButton1, //!< The first extra mouse button
-    XButton2, //!< The second extra mouse button
```

Comment:
Would it make sense to rename these enumerations to `X1` and `X2`?

Reply:
Extra1 and Extra2 would be more meaningful.

Reply:
I like that too. I'd happily approve a PR with that change.

Reply:
Here we go: #2838